doc/api.rst

```diff
@@ -57,6 +57,7 @@ Contains functions to extract features from `~mne.preprocessing.ICA` instance an
 
    get_topomap_array
    get_topomaps
+   get_psds
```

Comment:
Do we want a `get_psds_array`

Reply:
I am more inclined to make `get_topomap_array` private and remove it from the documentation. `get_topomaps` computes the topographic map one by one by calling `get_topomap_array`.
With the PSD feature, this is not needed and all PSDs can be computed at once. 

I don't see the added value of a function that can take a single IC as a numpy array and compute the corresponding feature. And supporting only the MNE API (with Raw|Epochs|ICA) gives us better control over what is provided as input to those functions than a numpy array.

Reply:
Okay agreed.

Reply:
@anandsaini024 shall you then make the `get_topomap_array` a private function and remove it from `api.rst`

Reply:
Done.